test/Validator/Text.php

```diff
@@ -78,8 +78,19 @@
                 (new Text)
                     ->email()
                     ->validate('test.com')
-                    ->errors()
-            )->toBe(['not an email']);
+                    ->errors()[0]
+                    ->__toString()
+            )->toBe('not an email');
+        });
+
+        it('can be translated', function () {
+            expect(
+                (new Text)
+                    ->email()
+                    ->validate('test.com')
+                    ->errors()[0]
+                    ->translate('hello')
```

Comment:
AH, or is this function going to take the token (`NOT_EMAIL`) _and_ the translation?